packages/config-utils/standalone/services/default/entitlements.js

```diff
@@ -19,17 +19,25 @@ module.exports = ({ env }) => ({
         'entitlements-config': `https://github.com/redhatinsights/entitlements-config#${getEntitlementsBranch(env)}`
     },
     register({ app, config }) {
-      app.get('/api/entitlements/v1/services', (_req, res) => {
-          const configPath = path.join(config.entitlements.assets['entitlements-config'], '/configs/bundles.yml');
-          // Use { json: true } in case of duplicate keys
-          const serviceSKUs = yaml.load(fs.readFileSync(configPath, 'utf8'), { json: true });
-          const services = serviceSKUs.map(serviceSKU => serviceSKU.name);
-          // Grant access to all services
-          const entitlements = services.reduce((acc, cur) => {
-              acc[cur] = { is_entitled: true, is_trial: false };
-              return acc;
-          }, {});
-          res.json(entitlements);
-      });
-    }
+        app.get('/api/entitlements/v1/services', (_req, res) => {
+            try {
+                const configPath = path.join(config.entitlements.assets['entitlements-config'], '/configs/stage/bundles.yml');
+
+                // Use { json: true } in case of duplicate keys
+                const outerYaml = yaml.load(fs.readFileSync(configPath, 'utf8'), { json: true });
+                const serviceSKUs = yaml.load(outerYaml.objects[0].data['bundles.yml'], { json: true });
+
+                const services = serviceSKUs.map(serviceSKU => serviceSKU.name);
+                // Grant access to all services
+                const entitlements = services.reduce((acc, cur) => {
+                    acc[cur] = { is_entitled: true, is_trial: false };
```

Comment:
We might also want to...

```suggestion
                    acc[cur] = { is_entitled: true };
```

to align with https://github.com/RedHatInsights/entitlements-config/pull/52

Reply:
I will be watching the PR :+1: